nebula_light_device.yaml

```diff
@@ -0,0 +1,283 @@
+# Refs.:
+# - https://github.com/M4GNV5/esphome-SK20-Nebula-Light
+# - https://github.com/fonix232/esphome_nebula_light
+# - https://github.com/kireque/esphome_nebula_light/issues/8
+
+bk72xx:
+  board: wb3s
```

Comment:
Since this is going to be a `package`, I'd recommend you strip out ALL the bits and pieces that aren't directly related to the _device_ definition (so no chip config, no nothing, just the light/fan definitions, and the outputs they rely on, and the globals)

Reply:
Better? I have moved the chip config into separate package and added it to `example.yaml` and to README file.

Reply:
Yes, it's nearly perfect - just don't forget to add `---` to the top of the package YAML to indicate it's a partial file!

Reply:
resolved in 9635c11e5974b7e6f93f294daea0f626baeac6d5